elementary/monitor/alerts/malformed.py

```diff
@@ -17,3 +18,11 @@ def to_slack(self, is_slack_workflow: bool = False) -> SlackMessageSchema:
                 f"```{json.dumps(self.data, indent=2)}```"
             )
         )
+
+    # We use getattribute and not getattr because "Alert" set some attributes to None if not given which skip the self.data.get(_name)
+    # For example - tags is set to None, so getattr for tags will return None although data contains it.
+    def __getattribute__(self, __name: str) -> Any:
+        try:
+            return super().__getattribute__(__name) or self.data.get(__name)
```

Comment:
In case we will pass other attributes then id and data that are were inheritance from the alert class 🙂 
I want it to be a fall back when the attributes are not define

Reply:
So you need to reverse the order in that case.
Otherwise it'll just crash.

```py
In [10]: class EGK:
    ...:     pass
    ...:

In [11]: x = EGK()

In [12]: x.__getattribute__("a")
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[12], line 1
----> 1 x.__getattribute__("a")

AttributeError: 'EGK' object has no attribute 'a'
```

Reply:
This is why I have a try exception :)
I try to get the attribute, if it is None I try and extract it from data, and if it is crashing I try and extract it from data

Reply:
The attribute can be none because default values of the Alert class.
So tags can be defined under data but still be set to None as the attribute.
So if we get None from the attribute, we want to search for it at the data

Reply:
Got it, thanks for explaining.
By the way, why not only search the `data`?
The `.owners` of the alert is constructed from `data` either way.